core/src/main/java/org/apache/carbondata/core/readcommitter/TableStatusReadCommittedScope.java

```diff
@@ -86,6 +87,13 @@ public TableStatusReadCommittedScope(AbsoluteTableIdentifier identifier,
     } else {
       SegmentFileStore fileStore =
           new SegmentFileStore(identifier.getTablePath(), segment.getSegmentFileName());
+      Optional<SegmentFileStore.FolderDetails>
```

Comment:
please add a comment, about this scenario

Reply:
removed this part, as we are setting the path in loadMetaDataDetails, it's no longer required.